providers/intercom/intercom.go

```diff
@@ -118,6 +120,10 @@ func userFromReader(reader io.Reader, user *goth.User) error {
 		return err
 	}
 
+	if u.EmailVerified != true {
+		return errors.New("User email not verified")
```

Comment:
By returning an error here, you're breaking existing functionality. Plus, it would be better to let the apps decide to do with a user that is unverified than to an error.